resources/deb/etc/init.d/openhab2

```diff
@@ -71,15 +71,36 @@ fi
 # Load the VERBOSE setting and other rcS variables
 . /lib/init/vars.sh
 
+# Check to see if openHAB is currently running
+EXISTINGPID=`pgrep java -G openhab | tail -1`
```

Comment:
I think we can not rely on pgrep being available on every debian/ubuntu installation from debian6, raspbian to current ubuntu, can we? A good hint is to check if other system services use this tool. If not, we shouldn't use it and have to stick with the basic unix/linux tools and the tools from init-functions.